pom.xml

```diff
@@ -664,7 +671,6 @@ THE SOFTWARE.
                 <bannedDependencies>
                   <excludes>
                     <exclude>org.sonatype.sisu:sisu-guice</exclude>
-                    <exclude>log4j:log4j</exclude>
```

Comment:
this should be here as well as the banning of commons-logging.
otherwise plugins can bundle the library by overriding the scope - which they may already have done (which is just a warning - rather than a build breakage)